### PR TITLE
Restore XML formatting for tool schema document.

### PR DESCRIPTION
**Pull request by jmchilton**
Brokenish with https://github.com/galaxyproject/galaxy/pull/3086 - at least with the two versions of Sphinx I tried.

### CODE REVIEW

**Comment by nsoranzo** [comment]
If anyone else is wondering what's the difference between code and code-block in rst files: https://stackoverflow.com/questions/34845889/whats-the-difference-between-the-code-and-code-block-directives-in-rest

File: doc/Makefile [hunk 1]
```diff
@@ -49,17 +49,20 @@ schema.md: parse_gx_xsd.py schema_template.md ../lib/galaxy/tools/xsd/galaxy.xsd
...
 	-rm -rf $(BUILDDIR)/* $(UPDATEWORKDIR) schema.md $(GENERATED_RST)
 
-html: $(GENERATED_RST)
+_html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
+html: $(GENERATED_RST) _html
+
```
Comment (nsoranzo) [inline doc/Makefile]: I don't understand the changes to the `html` target, why is that needed?

Also, you'd need to add `_html` to the .PHONY targets.

**Comment by nsoranzo** [comment]
Thanks @jmchilton!

File: doc/Makefile (hunk 1 above)
Comment (jmchilton) [reply doc/Makefile]: I reverted this part - I like that pattern because it allows me to test targets one at a time but maybe there is a better way to debug Makefile stuff?

Comment (nsoranzo) [reply doc/Makefile]: Debug Makefiles, is there such a thing? :laughing:
